charts/hivemq-platform/tests/services/hivemq_metrics_services_test.yaml

```diff
@@ -1043,7 +1043,7 @@ tests:
           hivemqProxyProtocol: true
     asserts:
       - failedTemplate:
-          errorMessage: Service type metrics with container port 9399 is using PROXY protocol value. PROXY protocol is only supported for MQTT and WebSocket services
+          errorPattern: Service type `metrics` with container port `9399` is using `hivemqProxyProtocol` value. HiveMQ PROXY protocol is only supported by MQTT and WebSocket services
```

Comment:
Question about the use of container port in this message.

Reply:
Same comment is applicable here: https://github.com/hivemq/helm-charts/pull/403#discussion_r1866225099